docs/dunst.pod

```diff
@@ -351,6 +351,10 @@ If it's set to false, long lines will be truncated an ellipsised.
 If it's set to true, long lines will be broken into multiple lines expanding
 the notification window height as necessary for them to fit.
 
+=item B<ellipsize> (values: [start/middle/end], default: middle)
+
+Specifies where truncated lines should be ellipsized.
```

Comment:
Could you please add also the information, that it's only used, if word_wrap is set to no? Also it would be great, if you add it to the dunstrc file, too.